codemagic.yaml

```diff
@@ -0,0 +1,99 @@
+definitions:
+  configure_environment: &configure_environment
+    instance_type: mac_mini_m1
+    max_build_duration: 60
```

Comment:
@lydiasama We usually use `30` mins timeout for the other scripts. Should we update this one to make it consistent? 🤔

Reply:
@doannimble I'm unaware we use `30` minutes in other scripts. 😂  Let me update. 👍🏻